images/galaxy-minimal/stable/Containerfile.core

```diff
@@ -5,8 +5,8 @@ ARG PULPCORE_VERSION=""
 ARG GALAXY_NG_VERSION=""
 
 RUN pip3 install --upgrade \
-  pulpcore${PULPCORE_VERSION} \
-  galaxy-ng${GALAXY_NG_VERSION} && \
+  galaxy-ng${GALAXY_NG_VERSION} \
+  pulpcore${PULPCORE_VERSION} && \
```

Comment:
I see you switched the order around. Does this fix something?

Reply:
Yeah, `pip` dependency resolver somehow fails, even when it shouldn't, when pulpcore is listed first. Since pulpcore is a dependency of galaxy it makes sense to let `pip` try to install galaxy first and then see if the requested pulpcore is compatible.